api/src/db/migrations/v108_add_business_name_search_to_formation_data.test.ts

```diff
@@ -0,0 +1,17 @@
+import { generateV107UserData } from "./v107_refactor_interstate_transport_essential_question";
+import { migrate_v107_to_v108 } from "./v108_add_business_name_search_to_formation_data";
+
+describe("migrate_v107_to_v108", () => {
+  it("adds the businessNameSearch field to formation data", () => {
+    const v107 = generateV107UserData({});
+    const v108 = migrate_v107_to_v108(v107);
+    expect(v108).toEqual({
+      ...v107,
+      formationData: {
+        ...v107.formationData,
+        businessNameSearch: undefined,
+      },
+      version: 108,
```

Comment:
There's no issue here, but as a note for the future, if there's no logic in your migration, then it doesn't actually need a test. Since this test is just a copy-paste of what the actual code does, it doesn't add any additional value.

It's not an issue here, of course. But yeah, we don't need to test if there's no conditional or no logic at all